<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainForm</class>
 <widget class="QMainWindow" name="mainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raƒçunalo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditKalk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>350</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>14</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>185</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>185</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeli">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>185</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMnozi">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlus">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDecimal">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMinus">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonZero">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>165</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRacunaj">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>255</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPaste">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCopy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutMemory">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveMemory">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>367</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
